transfer_queue/storage/clients/ray_storage_client.py

```diff
@@ -0,0 +1,126 @@
+from typing import Any
+
+import ray
+import torch
+from torch import Tensor
+
+from transfer_queue.storage.clients.base import TransferQueueStorageKVClient
+from transfer_queue.storage.clients.factory import StorageClientFactory
+
+@ray.remote
```

Comment:
this class will be accessed by many processes. consider use max_concurreny, like
```python
@ray.remote(max_concurrency=8)
```

Reply:
it enables multi-threading